pom.xml

```diff
@@ -87,32 +81,24 @@
             <artifactId>sshd-scp</artifactId>
             <version>2.8.0</version>
         </dependency>
-
+        <dependency>
```

Comment:
If maverick is the only ssh lib that is used please remove the other ssh dependencies and any other un-used dependenices.

Reply:
Done

Reply:
Please commit & push your changes to proceed with the PR